cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -259,6 +259,11 @@ func (m *AwsManager) buildNodeFromTemplate(asg *asg, template *asgTemplate) (*ap
 		}
 	}
 
+	resourcesFromTags := extractAllocatableResourcesFromAsg(template.Tags)
+	if val, ok := resourcesFromTags["ephemeral-storage"]; ok {
+		node.Status.Capacity[apiv1.ResourceEphemeralStorage] = *val
```

Comment:
Please update `Allocatable` as well (otherwise pods still won't be scheduled).

Reply:
Here's how it looks in the 1.14 branch: https://github.com/zalando-incubator/autoscaler/blob/zalando-cluster-autoscaler/cluster-autoscaler/cloudprovider/aws/aws_manager.go#L400

Reply:
Done.